docs/api/screen.md

```diff
@@ -14,7 +14,7 @@ property, so writing `let { screen } = require('electron')` will not work.
 
 An example of creating a window that fills the whole screen:
 
-```javascript
+```javascript fiddle='electron/master/docs/fiddles/screen/fit-screen'
```

Comment:
Hm, at scale this is gonna suck because it means when we make a version branch we have to adjust all these strings from `master` to `my-version`.  Can't we make it so that the website repo infers the `master` part and replaces it with the commit SHA the docs were pulled from?

Reply:
you mean pulling it from:
![image](https://user-images.githubusercontent.com/8472721/63061705-f07dba00-beaa-11e9-8a13-63d02934e577.png)

and just write `electron/docs/fiddles/screen/..`

Reply:
Personally I'd just put it as 

```
javascript fiddle='docs/fiddles/screen/fit-screen'
```

and infer the version + put the `electron` there.  i.e. `electron/{version}/{path}`

We shouldn't ever put a `gist` fiddle in the docs because then it isn't versioned and can be manipulated outside of the PR review process

Reply:
then how about also getting rid of the `docs/fiddles` part 🤔 and just write `fiddle='screen/fit-screen'`

Reply:
One day we might want to change that path or put fiddles in "guides" it should allow us to do that still IMO

Reply:
i mean it's not a big deal 😆 - so changing the root to be `e/e` and you'll give me green light 🍏?

Reply:
Haha this has descended into violent amounts of future-proofing 😆 

Technically I've already approved 😆 but I'll be 100% happy if the root is the root of `e/e`